docs/release-notes.md

```diff
@@ -33,6 +33,36 @@ n8n uses [semantic versioning](https://semver.org/){:target=_blank .external-lin
 You can find the release notes for older versions of n8n [here](/release-notes/0-x)
 ///
 
+## n8n@1.60.0
+
+View the [commits](https://github.com/n8n-io/n8n/compare/n8n@1.59.3...n8n@1.60.0){:target=_blank .external-link} for this version.<br />
+**Release date:** 2024-09-18
+
+This release contains new features, node enhancements and bug fixes.
+
+<div class="n8n-new-features" markdown>
+
+#### Queue metrics for workers
+
+You can now [expose and consume metrics from your workers](https://docs.n8n.io/hosting/configuration/configuration-examples/prometheus/). The worker instances have the same metrics available as the main instance(s) and can be configured with [environment variables](/hosting/configuration/environment-variables/endpoints/). 
+
+</div>
+
+You can now customize the maximum file size when uploading files within forms to webhooks. The [environment variable to set](/hosting/configuration/environment-variables/endpoints/) for this is `N8N_FORMDATA_FILE_SIZE_MAX`. The default setting is 200MiB.
+
+### Node updates
+Enhanced nodes: 
+
+- [Invoice Ninja](/integrations/builtin/app-nodes/n8n-nodes-base.invoiceninja/): Added actions for bank transactions
+- [OpenAI](integrations/builtin/app-nodes/n8n-nodes-langchain.openai/): Added O1 models to the model select
```

Comment:
This link is broken for me. Maybe missing an initial slash?

Reply:
Yes, it does. Let me fix that.